integration/walletextension/wallet_extension_test.go

```diff
@@ -385,7 +399,7 @@ func waitForWalletExtension(walletExtensionAddr string) error {
 }
 
 // Makes an Ethereum JSON RPC request and returns the response body.
-func makeEthJSONReq(walletExtensionAddr string, method string, params interface{}) []byte {
```

Comment:
This first param wasn't needed, because we use a file constant for the WE address.